container-storage-setup.sh

```diff
@@ -1814,6 +1841,19 @@ reset_storage() {
     fi
   fi
 
+  # If we created a volume group, remove volume group.
+  if [ "$_M_VG_CREATED" == "1" ];then
+    if ! remove_vg_if_exists "$_M_VG"; then
+      Error "Failed to remove volume group $_M_VG"
+      return 1
+    fi
+
+    # Cleanup any disks we added to volume group.
+    if ! remove_disk_pvs_parts "$_M_DEVS_RESOLVED";then
```

Comment:
I would let errors go to stderr.  Otherwise you are going to get a failure and have no idea why.